lib/applyFrame.php

```diff
@@ -3,8 +3,6 @@
 function applyFrame($sourceResource, $framePath) {
     $frame = imagecreatefrompng($framePath);
     $frame = resizePngImage($frame, imagesx($sourceResource), imagesy($sourceResource));
-    $x = imagesx($sourceResource) / 2 - imagesx($frame) / 2;
-    $y = imagesy($sourceResource) / 2 - imagesy($frame) / 2;
-    imagecopy($sourceResource, $frame, $x, $y, 0, 0, imagesx($frame), imagesy($frame));
+    imagecopy($sourceResource, $frame, 0, 0, 0, 0, imagesx($frame), imagesy($frame));
```

Comment:
as the dimensions of $frame are just defined as the dimensions of the source the $x and $y will always be 0.